OsmAnd/src/net/osmand/plus/plugins/odb/VehicleMetricsPlugin.kt

```diff
@@ -139,6 +139,21 @@ class VehicleMetricsPlugin(app: OsmandApplication) : OsmandPlugin(app), OBDReadS
 		val engineCoolantTempWidget: MapWidget =
 			createMapWidgetForParams(mapActivity, WidgetType.OBD_ENGINE_COOLANT_TEMP)
 		widgetsInfos.add(creator.createWidgetInfo(engineCoolantTempWidget))
+		val engineRuntimeWidget: MapWidget =
```

Comment:
Can we make widgets dynamic here based on WidgetType? add something like WidgetType.getObdTypes()